### ISSUE TITLE
2nd comskip file just keeps saying "Waiting?"

### ISSUE THREAD

**Issue by famewolf** [chacawaca/docker-post-recording#24]
A file has been processing for 2 1/2 hours now where typically it takes about 7 minutes.  When checking inside the container the first comskip file looks normal...the 2nd one however just keeps saying Waiting over and over.  Waiting for what?  The gpu?

Waiting
Waiting
Waiting
Waiting
Waiting
Waiting
Waiting
Waiting
Waiting
Waiting
Waiting
Waiting
Waiting

**Comment by famewolf** [comment]
The reason was a .lock file that was either there from a previous attempt by the same container.   I'd say this is still something that needs to be checked as if it tracks it's own attempts it should have been able to clear the lock.

**Comment by vdrover** [comment]
7 minutes is typical? Wow. That’s insanely fast. 

I run a cleanup script for this image that deletes any of temp folders it creates that are older then 2 days. This sometimes happens if I delete the source file before the task is finished. You could do something similar for lock files.

**Comment by famewolf** [comment]
It was an SD content movie so yeah 7 minutes.  1080p and up around 20 minutes...thats strictly for the encode (using fileflows) and not including comskip time.  I'm trying to tweak the custom.sh still to get things going right...the .ts files I have are going to be one of two types....one is a .ts that has a mpeg2 video from an hdhomerun tuner with dolby ac-3 audio.......the other is a recording of streamed content...it's a .ts that is in h264 with aac audio....I don't really see the need to reencode these but I do want to cut any commercials if they happen to have them.   Is there a way to differentiate them?  I'm a ffmpeg novice.  The movies it has try to transcode that were already h264 into hevc are playing VERY jerkily..enough to give me vertigo watching them. ;)

**Comment by vdrover** [comment]
I don’t think you can differentiate them. And you also can’t use a different config file based on folder location for example. 

Also, most users are leaving the commercials in place, and only skipping them. Commercial detection, in my experience, simply isn’t reliable enough to discard frames.

**Comment by famewolf** [comment]
I've been debating going to comchap but I have had ok results cutting them for my OTA hdhomerun mpeg files using logo detection as typically the commercials don't have the logo although you may have to adjust a few seconds if the logo fades out before the commercial starts etc.   Sadly though every file it attempts end up showing this:

2024-04-07 22:35:46.070846-04:00[autovideoconverter] Starting conversion of '/watch/movies/The Mortal Instruments (2013)/The Mortal Instruments (2013).ts'...
2024-04-07 22:35:46.076593-04:00[autovideoconverter] -------------------------------------
2024-04-07 22:35:46.078010-04:00[autovideoconverter] COMCUT START
2024-04-07 22:35:46.079389-04:00[autovideoconverter] COPY FILE TO TEMP DIRECTORY
2024-04-07 22:47:10.320355-04:00[autovideoconverter] COMCUT FINISH WITH STATUS 0
2024-04-07 22:47:10.321142-04:00[autovideoconverter] -------------------------------------
2024-04-07 22:47:10.322200-04:00[autovideoconverter] CCEXTRACTOR START
2024-04-07 22:47:47.346569-04:00[autovideoconverter] CCEXTRATOR FINISH WITH STATUS 10
2024-04-07 22:47:47.347766-04:00[autovideoconverter] -------------------------------------
2024-04-07 22:47:47.348813-04:00[autovideoconverter] FFMPEG START
2024-04-07 22:59:28.250916-04:00[autovideoconverter] FFMPEG FINISH WITH STATUS 0
2024-04-07 22:59:28.252052-04:00[autovideoconverter] -------------------------------------
2024-04-07 22:59:28.255186-04:00[autovideoconverter] MOVE TMP FILE TO DESTINATION FOLDER
2024-04-07 22:59:28.257977-04:00[autovideoconverter] **Comskip failed.**    <------------------
2024-04-07 22:59:28.261484-04:00[autovideoconverter] CCextrator failed.
2024-04-07 22:59:28.262916-04:00[autovideoconverter] -------------------------------------
2024-04-07 22:59:28.267570-04:00[autovideoconverter] TMP DIRECTORY REMOVED
2024-04-07 22:59:28.268896-04:00[autovideoconverter]  
2024-04-07 22:59:28.270023-04:00[autovideoconverter] =======================================     CONVERSION DONE     =================================================
2024-04-07 22:59:28.271098-04:00[autovideoconverter] Conversion ended successfully.
2024-04-07 22:59:28.272480-04:00[autovideoconverter] =========================================================================================================


Also when the script checks:   if [ $codec_name = "MPE" ] ; then 

I think that could be modified to check for MPEG2 specifically vs MPEG-4...gonna have to wait till my next hdhomerun recording occurs.
